swap_meet/vendor.py

```diff
@@ -1,2 +1,24 @@
+from .item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory
+
+    def add(self, item):
+        updated_inventory = self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            updated_inventory = self.inventory.remove(item)
+        else: 
+            item = False
+        return item
```

Comment:
Consider refactoring to first check if item is not available. This design is called a guard clause, and can be a nice pattern to follow to enhance readability.
```suggestion
        if item not in self.inventory:
            return False
            
        updated_inventory = self.inventory.remove(item)
        return item
```